US-CBP/cbp-theme
# Issue75

#### What's this PR do?
This is addressing Issue#75, by adding an input type file element with default button styling.

#### Where should the reviewer start?
Look on the theme index file around line 1808 for the new form element

#### How should this be manually tested?
use all 3 major browsers to check the new "Choose a file" input to make sure it is accessible by keyboard, and shows the .btn-default styling for :focus and :hover events.

#### Any background context you want to provide?
For helping with Issue #75 

#### What are the relevant tickets?
n/a
#### Screenshots (if appropriate)

![image]( https://user-images.githubusercontent.com/24257663/55184814-7baf1b00-5169-11e9-96f2-c6e4936b80ba.png )


#### Questions:
- Is there a blog post?
https://github.com/US-CBP/cbp-theme/issues/75

- Does the knowledge base need an update?
